tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+# Arrange
+import pytest
+from pathlib import Path
+@pytest.fixture
```

Comment:
```suggestion


@pytest.fixture
```

Once we have ruff'ed the codebase we should add some pre-commit hooks to ensure the formatting is always applied